releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -2,6 +2,17 @@
 
 [Source code][source_code] | [Package (Maven)][package] | [Product documentation][docs] | [Samples][samples]
 
+# spring-cloud-azure-feature-management-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 1.1.2/1.2.2 - February 24, 2020
+
+* AppConfigurationRefresh's refreshConfigurations() can be called to check if the configurations are not up to date and if the cache has expired trigger a refresh event.
+* A new authentication method has been added. AppConfigurationCredentialProvider and KeyVaultCredentialProvider can be used to pass TokenCredentials to the provider to use when connecting with App Configuration and Key Vault. This method also allows for different authentication methods/credentials to be used when connecting with multiple App Configuration stores or Key Vaults.
```

Comment:
We are NOT adding a new authentication method. We just added a way for users to provide custom `TokenCredential` for AAD authentication.